HW_1/Task_2.py

```diff
@@ -0,0 +1,19 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# Пример: 123 -> 6 (1 + 2 + 3) 100 -> 1 (1 + 0 + 0)
+
+# 1 вариант решения задачи
+# n = input("Введите трехзначное число: ")
+# a = int(n[0])
+# b = int(n[1])
+# c = int(n[2])
+# print("Сумма трехзначнего числа равна:", a + b + c)
+
+# 2 вариант решения данной задачи
+
+n = input("Введите трехзначное число: ")
+n = int(n)
```

Comment:
Это можно совместить
```suggestion
n = int(input("Введите трехзначное число: "))
```